package.json

```diff
@@ -30,21 +31,21 @@
   },
   "homepage": "https://github.com/chartjs/chartjs-test-utils#readme",
   "devDependencies": {
-    "@rollup/plugin-commonjs": "^20.0.0",
-    "@rollup/plugin-node-resolve": "^13.0.0",
-    "eslint": "^7.20.0",
+    "@rollup/plugin-commonjs": "^22.0.2",
+    "@rollup/plugin-node-resolve": "^13.3.0",
+    "eslint": "^8.21.0",
     "eslint-config-chartjs": "^0.3.0",
     "eslint-plugin-es": "^4.1.0",
     "esm": "^3.2.25",
-    "mocha": "^8.3.0",
-    "rollup": "^2.39.0"
+    "mocha": "^10.0.0",
+    "rollup": "^2.77.2"
   },
   "dependencies": {
-    "pixelmatch": "^5.2.1"
+    "pixelmatch": "^5.3.0"
   },
   "peerDependencies": {
-    "jasmine": "^3.6.4",
-    "karma": "^6.1.1",
-    "karma-jasmine": "^4.0.1"
+    "jasmine": ">=3.0.0",
+    "karma": ">=6.0.0",
+    "karma-jasmine": ">=4.0.0"
```

Comment:
Shouldn't we keep the minimum versions like they are now?

Reply:
The prior minimum versions aren't actually the minimal accepted versions but were just copied over from what was previously in `dependencies`: https://github.com/chartjs/chartjs-test-utils/pull/18. So lower versions should work and there's no reason to disallow them. That being said, no one has complained, and I have no objection to leaving the minimum version at a higher value if you'd still like to do that

Reply:
dont know if we use any things that got addeded or fixed between what we used before and the lower version now, if thats not the case this is fine